### ISSUE TITLE
Unexpected number of children

### ISSUE THREAD

**Issue by djberg96** [mvz/ripper_ruby_parser#64]
ruby 2.4.5p335 (2018-10-18 revision 65137) [x86_64-linux]

After upgrading to version 1.5.0 in my local dev environment and attempting to run our specs, I'm hitting `RuntimeError: Unexpected number of children: 1`. The file in mentioned in the backtrace is https://github.com/ManageIQ/manageiq/blob/master/app/models/authenticator.rb.

You can duplicate it via:

```
file = 'app/models/authenticator.rb'
content = File.read(flie)
RipperRubyParser::Parser.new.parse(content, file)
```

Full backtrace:

```
RuntimeError: Unexpected number of children: 1
	from /home/dberger/.rbenv/versions/2.4.5/lib/ruby/gems/2.4.0/gems/ripper_ruby_parser-1.5.0/lib/ripper_ruby_parser/sexp_handlers/helper_methods.rb:14:in `extract_node_symbol'
	from /home/dberger/.rbenv/versions/2.4.5/lib/ruby/gems/2.4.0/gems/ripper_ruby_parser-1.5.0/lib/ripper_ruby_parser/sexp_handlers/methods.rb:32:in `process_defs'
	from /home/dberger/.rbenv/versions/2.4.5/lib/ruby/gems/2.4.0/gems/sexp_processor-4.10.1/lib/sexp_processor.rb:269:in `block (2 levels) in process'
	from /home/dberger/.rbenv/versions/2.4.5/lib/ruby/gems/2.4.0/gems/sexp_processor-4.10.1/lib/sexp_processor.rb:324:in `error_handler'
	from /home/dberger/.rbenv/versions/2.4.5/lib/ruby/gems/2.4.0/gems/sexp_processor-4.10.1/lib/sexp_processor.rb:268:in `block in process'
	from /home/dberger/.rbenv/versions/2.4.5/lib/ruby/gems/2.4.0/gems/sexp_processor-4.10.1/lib/sexp_processor.rb:388:in `in_context'
	from /home/dberger/.rbenv/versions/2.4.5/lib/ruby/gems/2.4.0/gems/sexp_processor-4.10.1/lib/sexp_processor.rb:245:in `process'
	from /home/dberger/.rbenv/versions/2.4.5/lib/ruby/gems/2.4.0/gems/ripper_ruby_parser-1.5.0/lib/ripper_ruby_parser/sexp_processor.rb:145:in `process_comment'
	from /home/dberger/.rbenv/versions/2.4.5/lib/ruby/gems/2.4.0/gems/sexp_processor-4.10.1/lib/sexp_processor.rb:269:in `block (2 levels) in process'
	from /home/dberger/.rbenv/versions/2.4.5/lib/ruby/gems/2.4.0/gems/sexp_processor-4.10.1/lib/sexp_processor.rb:324:in `error_handler'
	from /home/dberger/.rbenv/versions/2.4.5/lib/ruby/gems/2.4.0/gems/sexp_processor-4.10.1/lib/sexp_processor.rb:268:in `block in process'
	from /home/dberger/.rbenv/versions/2.4.5/lib/ruby/gems/2.4.0/gems/sexp_processor-4.10.1/lib/sexp_processor.rb:388:in `in_context'
	from /home/dberger/.rbenv/versions/2.4.5/lib/ruby/gems/2.4.0/gems/sexp_processor-4.10.1/lib/sexp_processor.rb:245:in `process'
	from /home/dberger/.rbenv/versions/2.4.5/lib/ruby/gems/2.4.0/gems/ripper_ruby_parser-1.5.0/lib/ripper_ruby_parser/sexp_handlers/helper_methods.rb:56:in `block in map_process_list'
	from /home/dberger/.rbenv/versions/2.4.5/lib/ruby/gems/2.4.0/gems/sexp_processor-4.10.1/lib/sexp.rb:65:in `map'
	from /home/dberger/.rbenv/versions/2.4.5/lib/ruby/gems/2.4.0/gems/sexp_processor-4.10.1/lib/sexp.rb:65:in `map'
... 24 levels...
	from /home/dberger/.rbenv/versions/2.4.5/lib/ruby/gems/2.4.0/gems/ripper_ruby_parser-1.5.0/lib/ripper_ruby_parser/sexp_handlers/helper_methods.rb:52:in `map_process_list_compact'
	from /home/dberger/.rbenv/versions/2.4.5/lib/ruby/gems/2.4.0/gems/ripper_ruby_parser-1.5.0/lib/ripper_ruby_parser/sexp_processor.rb:78:in `process_stmts'
	from /home/dberger/.rbenv/versions/2.4.5/lib/ruby/gems/2.4.0/gems/sexp_processor-4.10.1/lib/sexp_processor.rb:269:in `block (2 levels) in process'
	from /home/dberger/.rbenv/versions/2.4.5/lib/ruby/gems/2.4.0/gems/sexp_processor-4.10.1/lib/sexp_processor.rb:324:in `error_handler'
	from /home/dberger/.rbenv/versions/2.4.5/lib/ruby/gems/2.4.0/gems/sexp_processor-4.10.1/lib/sexp_processor.rb:268:in `block in process'
	from /home/dberger/.rbenv/versions/2.4.5/lib/ruby/gems/2.4.0/gems/sexp_processor-4.10.1/lib/sexp_processor.rb:388:in `in_context'
	from /home/dberger/.rbenv/versions/2.4.5/lib/ruby/gems/2.4.0/gems/sexp_processor-4.10.1/lib/sexp_processor.rb:245:in `process'
	from /home/dberger/.rbenv/versions/2.4.5/lib/ruby/gems/2.4.0/gems/ripper_ruby_parser-1.5.0/lib/ripper_ruby_parser/sexp_processor.rb:53:in `process_program'
	from /home/dberger/.rbenv/versions/2.4.5/lib/ruby/gems/2.4.0/gems/sexp_processor-4.10.1/lib/sexp_processor.rb:269:in `block (2 levels) in process'
	from /home/dberger/.rbenv/versions/2.4.5/lib/ruby/gems/2.4.0/gems/sexp_processor-4.10.1/lib/sexp_processor.rb:324:in `error_handler'
	from /home/dberger/.rbenv/versions/2.4.5/lib/ruby/gems/2.4.0/gems/sexp_processor-4.10.1/lib/sexp_processor.rb:268:in `block in process'
	from /home/dberger/.rbenv/versions/2.4.5/lib/ruby/gems/2.4.0/gems/sexp_processor-4.10.1/lib/sexp_processor.rb:388:in `in_context'
	from /home/dberger/.rbenv/versions/2.4.5/lib/ruby/gems/2.4.0/gems/sexp_processor-4.10.1/lib/sexp_processor.rb:245:in `process'
	from /home/dberger/.rbenv/versions/2.4.5/lib/ruby/gems/2.4.0/gems/ripper_ruby_parser-1.5.0/lib/ripper_ruby_parser/parser.rb:21:in `parse'
	from (irb):5
	from /home/dberger/.rbenv/versions/2.4.5/bin/irb:11:in `<main>'
```

**Comment by mvz** [comment]
@djberg96 this is fixed in version 1.5.1.

**Comment by djberg96** [comment]
Thanks!
